mods/cnc/weapons/missiles.yaml

```diff
@@ -178,15 +122,11 @@ MammothMissiles:
 	Report: rocket1.aud
 	ValidTargets: Ground
 	Projectile: Bullet
-		Arm: 5
```

Comment:
Bleh, both our comments are invalid. Since 227mm rockets use the `Bullet` projectile, missile-specific properties are ignored completely.